### ISSUE TITLE
Papenmeier Braillex Trio not recognized under NVDA 2019.3

### ISSUE THREAD

**Issue by MichelSuch** [nvaccess/nvda#10780]
Someone reports me that his Braillex Trio is not recognized under 2019.3. He tried connection in USB or Bluetooth and got the same result.
### Steps to reproduce:
Connect a Braillex Trio to the PC, select it in the Braille Settings dialog.
### Actual behavior:
After selecting the braille terminal, NVDA says it cannot load the braille driver (see log attached)
### Expected behavior:
Should load
#### NVDA installed/portable/running from source:
Installed or portable, same problem
#### NVDA version:
2019.3 and 2019.3.1
#### Windows version:
W10 64 bits
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Works fine with 2019.2.1
[nvda.log](https://github.com/nvaccess/nvda/files/4190584/nvda.log)

**Comment by DrSooom** [comment]
Could you please try this with [NVDA 2019.3 Beta 1](https://ci.appveyor.com/api/buildjobs/ra7vb4b94dlutb9l/artifacts/output/nvda_2019.3beta1.exe) Portable?

**Comment by MichelSuch** [comment]
Some more information on this:
The problem seems to come from a conflict with brxcom, the software that allows tu use the terminal's keyboard with no screen reader active.
When brxcom is desinstalled, the display works fine, but only the display function, the keyboard remains inactive.

**Comment by MichelSuch** [comment]
Oops, did not want to close this one!

**Comment by leonardder** [comment]
This driver is pretty complicated and really needs a revisit. I wasn't able to test this with brxcom when I had a device available.

We really need a log with level debug, I think

**Comment by MichelSuch** [comment]
Hi,

Here is the piece of log I got from the user, hope it can help, I paste it here because it is short.
DEBUG - autoSettingsUtils.autoSettings.AutoSettings._registerConfigSaveAction (22:00:38.224) - MainThread (940):
registering pre_configSave action: <class 'brailleDisplayDrivers.papenmeier.BrailleDisplayDriver'>
DEBUGWARNING - brailleDisplayDrivers.papenmeier.BrailleDisplayDriver.connectBluetooth (22:00:38.535) - MainThread (940):
connectBluetooth failed
ERROR - braille.BrailleHandler.setDisplayByName (22:00:38.535) - MainThread (940):
Error initializing display driver papenmeier for kwargs {}
Traceback (most recent call last):
  File "braille.pyc", line 1725, in setDisplayByName
  File "brailleDisplayDrivers\papenmeier.pyc", line 306, in __init__
Exception: no device found

**Comment by hss16** [comment]
Please try to connect your Trio with usb. That works as expected. Except braille input.

**Comment by Adriani90** [comment]
@MichelSuch are you still having this issue with NVDA 2024.2 or 2024.3 Beta?
